### DISCUSSION TITLE
How/Can I get a debuggers beret

### DISCUSSION THREAD

**Discussion by Happy12222** [SmartlyDressedGames/Unturned-3.x-Community#4678] [Q&A] [answered]
When I first started to report bugs I was motivated by this beret but then a heard its actually unattainable. I liked finding bugs even without this beret but still very сurious if its even possible to get it.

**Comment by SDGNelson** [comment, marked as answer]
Unfortunately the Debugger's Beret (purple) has not been given out anymore for the several years. It became a problematic incentive encouraging players to look for issues rather than playing the game, and resulted in a lot of duplicate reports. Sorry to disappoint!

**Comment by Roadbobek** [comment]
But if someone finds a new bug will they still get it?

**Reply by Jdance-Media** [reply]
As Nelson put earlier, the beret also "became a problematic incentive encouraging players to look for issues rather than playing the game". That part of the reasoning is not directly related to duplicate reports but rather really small issues being found that don't really affect much.

Even if you find a relatively major bug, you won't receive the Debugger's Beret for it. However, you do normally get credited in the update notes for your work in finding the issue. With that being said, reporting major security issues with the game (e.g. Remote Code Execution) will result in you receiving a White Hat beret normally.
